src/EnergyPlus/BranchNodeConnections.cc

```diff
@@ -1673,17 +1657,9 @@ void GetChildrenData(EnergyPlusData &state,
     Array1D_string ChildOutNodeName;
     Array1D_int ChildInNodeNum;
     Array1D_int ChildOutNodeNum;
-    int CountNum;
     bool ErrInObject;
-    std::string MatchNodeName;
-    std::string ParentInletNodeName;
-    std::string ParentOutletNodeName;
-    int ParentInletNodeNum;
-    int ParentOutletNodeNum;
-    int CountMatchLoop;
-
-    for (auto &thisChildrenCType : ChildrenCType)
-        thisChildrenCType = DataLoopNode::ConnectionObjectType::Invalid;
+
+    std::fill(ChildrenCType.begin(), ChildrenCType.end(), DataLoopNode::ConnectionObjectType::Invalid);
```

Comment:
[src/EnergyPlus/BranchNodeConnections.cc:1686]:(style),[useStlAlgorithm],Consider using std::fill or std::generate algorithm instead of a raw loop.